daemon/src/model/cfd.rs

```diff
@@ -1685,6 +1955,9 @@ pub struct Cet {
 /// To use any of these transactions, we need to re-sign them with the correct secret key.
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
 pub struct Dlc {
+    // TODO: Add the oracle event id to the dlc?
```

Comment:
Might be a good idea actually.

Reply:
I can see to model it like that. One thing to keep in mind is that we use the event-id (in particular its timestamp) for calculations outside the DLC. I feel we might introduce a dependency here that we don't want.